storybook/native/main.js

```diff
@@ -14,6 +14,7 @@ const theme = {
 
 // import stories
 configure(() => {
+  require('../../src/atoms/Icon/Icon.stories');
```

Comment:
Can't you add a regex here? for all `**.stories.js`

Reply:
No :( 

We explored some other options like plugins to do this but nothing was good enough. We need to manually add each story until storybook adds proper configuration support like web